doc/devel/coding_guide.rst

```diff
@@ -47,6 +47,26 @@ PR Review guidelines
 * If you have commit rights, then you are trusted to use them.  Please
   help review and merge PRs!
 
+* If you have the rights to set labels, tag the PR with descriptive labels.
+  See the `list of labels <https://github.com/matplotlib/matplotlib/labels>`__.
+
+* Set the milestone according to these rules:
+
+  * *New features and API changes* are milestoned for the next point-release
+    ``v3.X.0``.
+
```

Comment:
The x numbering here is a bit confusing   Bug fixes are put to x-1.  Also someone will have to come back and change the 3.X to 4.X   Suggest just saying I.e. 3.3.0 and 3.2.7 to make clear they are just examples.

Reply:
@jklymak Thanks for the comments.

Not too much worried about 3.X vs. 4.X. Will be still ome time, and even then it's easily to translate.

The advatage of `v3.X.0` / `v3.X.Y` is that it emphasizes the trailing 0 vs. arbitrary number. That may be a bit harder to see in an explicit example.

I'd leave it like this unless more devs would prefer the explicit example.

Reply:
My point is that the bug-fix milestones are one below the point-release, i.e. 3.X.0 and 3.(X-1).Y

Reply:
That's technically true, but 3.(X-1).Y seems really more intimidating than helpful. The two mentions are in separate bullet points concerning different types of PRs. I don't think one has to assume that X is the same. Anyway, usually we'll have just one bugfix milestone at a time.

So, would also leave this ambiguity unless other devs would wish otherwise.